Domain/Models/Inventory.cs

```diff
@@ -0,0 +1,71 @@
+﻿using InventoryManagementSystem.Domain.General;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace InventoryManagementSystem.Domain.Models
```

Comment:
`Models` usually used to have the DTOs/entities. Inventory can go to a different folder `Services`